packages/stats/tests/bayesian/test_tests.py

```diff
@@ -156,5 +161,145 @@ def test_inexact_log_approximation(self):
+        )
+        q_stat_t = QuantileStatistic(
+            n=n_t,
+            nu=nu,
+            alpha=alpha,
+            quantile_hat=quantile_hat_t,
+            quantile_lower=quantile_lower_t,
+            quantile_upper=quantile_upper_t,
+            main_sum=main_sum_t,
+            denominator_sum=denominator_sum_t,
+            main_sum_squares=main_sum_squares_t,
+            denominator_sum_squares=denominator_sum_squares_t,
+            main_denominator_sum_product=denominator_sum_product_t,
+        )
+
+        b_flat = GaussianEffectABTest(
+            q_stat_c, q_stat_t, config=effect_config_flat
+        ).compute_result()
+        b_relative_flat = GaussianEffectABTest(
+            q_stat_c, q_stat_t, config=effect_config_flat_rel
+        ).compute_result()
+        b_informative = GaussianEffectABTest(
+            q_stat_c, q_stat_t, config=effect_config_inf
+        ).compute_result()
+        b_relative_informative = GaussianEffectABTest(
+            q_stat_c, q_stat_t, config=effect_config_inf_rel
+        ).compute_result()
+
+        self.assertEqual(b_flat.expected, 0.5365124367824539)
+        self.assertEqual(b_relative_flat.expected, 0.07495297222890572)
+        self.assertEqual(b_informative.expected, 0.5357380334109081)
+        self.assertEqual(b_relative_informative.expected, 0.07495191495960002)
+        self.assertEqual(b_flat.ci, [0.4572559582016913, 0.6157689153632164])
+        self.assertEqual(b_relative_flat.ci, [0.06341005842626224, 0.0864958860315492])
+        self.assertEqual(b_informative.ci, [0.4565471445922169, 0.6149289222295993])
+        self.assertEqual(
+            b_relative_informative.ci, [0.06340910198272585, 0.08649472793647417]
+        )
+
+    def test_bayesian_gaussian_ab_test_priors(self):
+        result = GaussianBayesianABTest(
+            SampleMeanStatistic(sum=100, sum_squares=1002.25, n=10),
+            SampleMeanStatistic(sum=105, sum_squares=1111.5, n=10),
+            config=GaussianBayesianConfig(
+                prior_a=GaussianPrior(0, 100, 1), prior_b=GaussianPrior(0, 20, 1)
+            ),
+        ).compute_result()
+
+        expected_rounded_dict = asdict(
+            BayesianTestResult(
+                expected=0.04974,
+                ci=[-0.01797, 0.12212],
+                uplift=Uplift(dist="lognormal", mean=0.04854, stddev=0.03402),
+                chance_to_win=0.9232,
+                risk=[0.51017, 0.01277],
+            )
+        )
+        result_rounded_dict = round_results_dict(asdict(result))
+        self.assertDictEqual(result_rounded_dict, expected_rounded_dict)
+
+    def test_missing_data(self):
+        result = GaussianBayesianABTest(
+            SampleMeanStatistic(sum=0, sum_squares=0, n=0),
+            SampleMeanStatistic(sum=0, sum_squares=0, n=0),
+        ).compute_result()
+        self.assertEqual(result.chance_to_win, 0.5)
+        self.assertEqual(result.expected, 0)
+
+    def test_inexact_log_approximation(self):
```

Comment:
Why do we need to replicate this test here? Maybe copy paste?

Reply:
good catch, i removed this